### ISSUE TITLE
Stop referencing the old Converter class in PropertyExpression

### ISSUE THREAD

**Issue by TheLimeGlass** [SkriptLang/Skript#6685]
### Suggestion

Stop referencing the old Converter class in PropertyExpression https://github.com/SkriptLang/Skript/blob/b0052d25d6314c0cf7279d089aa4201bad23f713/src/main/java/ch/njol/skript/expressions/base/PropertyExpression.java#L116

This restricts the usage of method references `Block::getExample()` by using the old converter class giving an IDE warning about using deprecated classes.

### Why?

To enforce the code standards the deprecation needs to be changed.

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by APickledWalrus** [comment]
It currently uses the old classes for compatibility reasons, see https://github.com/SkriptLang/Skript/pull/550

**Comment by TheLimeGlass** [comment]
> It currently uses the old classes for compatibility reasons, see #5507

Yes, but it needs to be replaced/overloaded with another method or finally removed.
